torchtext/vocab.py

```diff
@@ -553,9 +553,7 @@ def build_vocab_from_iterator(iterator):
     """
 
     counter = Counter()
-    with tqdm(unit_scale=0, unit='lines') as t:
```

Comment:
Can we move this into a separate PR? tqdm was used within the dataset building file as well, but this is a wide-ranging change for the vocab which is used in many places.

Reply:
Moved it back.